### ISSUE TITLE
Mutliple Critical and High Vulnerabilities in gcr.io/distroless/java:11

### ISSUE THREAD

**Issue by georgeherby** [GoogleContainerTools/distroless#962]
Tooks like libexpat1 need to be upgraded from `.2.10-2+deb11u1` to `2.2.10-2+deb11u2`

```
Total: 5 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 2, CRITICAL: 3)
+-----------+------------------+----------+-------------------+------------------+---------------------------------------+
|  LIBRARY  | VULNERABILITY ID | SEVERITY | INSTALLED VERSION |  FIXED VERSION   |                 TITLE                 |
+-----------+------------------+----------+-------------------+------------------+---------------------------------------+
| libexpat1 | CVE-2022-25235   | CRITICAL | 2.2.10-2+deb11u1  | 2.2.10-2+deb11u2 | expat: malformed 2- and               |
|           |                  |          |                   |                  | 3-byte UTF-8 sequences can            |
|           |                  |          |                   |                  | lead to arbitrary code...             |
|           |                  |          |                   |                  | -->avd.aquasec.com/nvd/cve-2022-25235 |
+           +------------------+          +                   +                  +---------------------------------------+
|           | CVE-2022-25236   |          |                   |                  | expat: namespace-separator characters |
|           |                  |          |                   |                  | in "xmlns[:prefix]" attribute         |
|           |                  |          |                   |                  | values can lead to arbitrary code...  |
|           |                  |          |                   |                  | -->avd.aquasec.com/nvd/cve-2022-25236 |
+           +------------------+          +                   +                  +---------------------------------------+
|           | CVE-2022-25315   |          |                   |                  | expat: integer overflow               |
|           |                  |          |                   |                  | in storeRawNames()                    |
|           |                  |          |                   |                  | -->avd.aquasec.com/nvd/cve-2022-25315 |
+           +------------------+----------+                   +                  +---------------------------------------+
|           | CVE-2022-25313   | HIGH     |                   |                  | expat: stack exhaustion               |
|           |                  |          |                   |                  | in doctype parsing                    |
|           |                  |          |                   |                  | -->avd.aquasec.com/nvd/cve-2022-25313 |
+           +------------------+          +                   +                  +---------------------------------------+
|           | CVE-2022-25314   |          |                   |                  | expat: integer overflow               |
|           |                  |          |                   |                  | in copyString()                       |
|           |                  |          |                   |                  | -->avd.aquasec.com/nvd/cve-2022-25314 |
+-----------+------------------+----------+-------------------+------------------+---------------------------------------+
```

**Comment by dinvlad** [comment]
Would it be possible to add a more regular cadence for upgrading base images automatically?

**Comment by imclem** [comment]
I'm interested in this fix too.

**Comment by dinvlad** [comment]
This `expat` issue is quite annoying because it keeps popping up everywhere. Might make sense to groom all of the Distroless images for it ;)

**Comment by loosebazooka** [comment]
It's on daily cadence. But I've manually triggered an update. If debian has updated it, we'll pick it up

**Comment by imclem** [comment]
@loosebazooka it's improved but we still see: `CVE-2022-25236` coming up in the report. Not sure why since it's still indicates that it's fixed in 2.2.10-2+deb11u2

```
=========================
Total: 2 (CRITICAL: 2)
+-----------+------------------+----------+-------------------+------------------+---------------------------------------+
|  LIBRARY  | VULNERABILITY ID | SEVERITY | INSTALLED VERSION |  FIXED VERSION   |                 TITLE                 |
+-----------+------------------+----------+-------------------+------------------+---------------------------------------+
| libexpat1 | CVE-2022-25236   | CRITICAL | 2.2.10-2+deb11u1  | 2.2.10-2+deb11u2 | expat: namespace-separator characters |
|           |                  |          |                   |                  | in "xmlns[:prefix]" attribute         |
|           |                  |          |                   |                  | values can lead to arbitrary code...  |
|           |                  |          |                   |                  | -->avd.aquasec.com/nvd/cve-2022-25236 |
+           +------------------+          +                   +                  +---------------------------------------+
|           | CVE-2022-25315   |          |                   |                  | expat: integer overflow               |
|           |                  |          |                   |                  | in storeRawNames()                    |
|           |                  |          |                   |                  | -->avd.aquasec.com/nvd/cve-2022-25315 |
+-----------+------------------+----------+-------------------+------------------+---------------------------------------+
```

**Comment by k0h1zuk4** [comment]
FYI: The Debian security tracker also says these issues are fixed in 2.2.10-2+deb11u2.

https://security-tracker.debian.org/tracker/CVE-2022-25236
https://security-tracker.debian.org/tracker/CVE-2022-25315

**Comment by david-creigh-msi** [comment]
Looks like this got fixed in #963 (at least it did for my organisation), are y'all still affected by this?

**Comment by imclem** [comment]
@david-creigh-msi ok on my side
